TheMovieDb/ViewModel/MovieViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MoviesViewModel.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 03/11/21.
+//
+
+import UIKit
+
+struct MovieViewModel {
+    
+    var movie: Movie
+    
+    var isHighSection = false
+    
+    var numerTop: Int?
```

Comment:
do we really need it as a var ?

Reply:
I leave the others variables as var because I'm using to show number rated in a cell and isHighSection to know if is the top view
```suggestion
        let movie: Movie
```